### DISCUSSION TITLE
Add Homebrew install info

### DISCUSSION THREAD

**Discussion by alebcay** [sayanarijit/xplr#72] [Show and tell]
On a similar note as #24, `xplr` can now be installed on Linux and macOS systems via the Homebrew package manager. For Homebrew users:

```
brew update
brew install xplr
```

**Comment by sayanarijit** [comment]
Thanks @alebcay :+1:

**Comment by sayanarijit** [comment]
Link to the formulae: [formulae.brew.sh/formula/xplr](https://formulae.brew.sh/formula/xplr)

**Comment by sayanarijit** [comment]
@alebcay There's a critical update: https://github.com/sayanarijit/xplr/releases/tag/v0.5.10
